### PR TITLE
Feature/contact page/94 add components styling page

### PR DESCRIPTION
**Pull request by lanniiee**
created contact content and edited the bankcontact component to implement API
https://trello.com/c/AlvEIq9i/94-create-a-contacts-page

### CODE REVIEW

File: src/pages/ContactList/ContactList.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import "./ContactList.scss";
+import ContactContent from "../../components/ContactContent/ContactContent";
+// import { getUserContacts } from "../../utils/apiUtils";
+// import { useState } from "react";
```
Comment (kirsty-nology) [inline src/pages/ContactList/ContactList.jsx]: remove comments

File: src/pages/ContactList/ContactList.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const ContactList = () => {
+
+// const [userData, setUserData] = UseState([]);
```
Comment (kirsty-nology) [inline src/pages/ContactList/ContactList.jsx]: remove comment

File: src/pages/ContactList/ContactList.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    return (
+        <div className="contactList">
+            <ContactContent className="contactList__main" />
```
Comment (kirsty-nology) [inline src/pages/ContactList/ContactList.jsx:8]: do not add a `className` like this, it will be read as a prop, also it's unnecessary

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

File: src/pages/Wallet/Wallet.jsx [hunk 4]
```diff
@@ -15,7 +14,6 @@ const Wallet = ({ amount = "0" }) => {
           <Funds amount={amount} isManage={true} />
         </div>
         <div className="wallet-ctner__contacts">
```
Comment (kirsty-nology) [inline src/pages/Wallet/Wallet.jsx:16]: container no longer relevant

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

File: src/pages/Wallet/Wallet.jsx (hunk 4 above)
Comment (lanniiee) [reply src/pages/Wallet/Wallet.jsx:16]: i think this div was suppose to be used when contact content component is done

**Review by stu-slicer — APPROVED** [review]

**Pull request merged**